lightning/src/ln/channelmanager.rs

```diff
@@ -8812,7 +8812,7 @@ macro_rules! create_offer_builder { ($self: ident, $builder: ty) => {
 		let path = $self.create_blinded_paths_using_absolute_expiry(context, absolute_expiry)
 			.and_then(|paths| paths.into_iter().next().ok_or(()))
 			.map_err(|_| Bolt12SemanticError::MissingPaths)?;
-		let builder = OfferBuilder::deriving_signing_pubkey(node_id, expanded_key, nonce, secp_ctx)
+		let builder = OfferBuilder::deriving_issuer_id(node_id, expanded_key, nonce, secp_ctx)
```

Comment:
I assume that this is a would-like-to-have review on Bolts https://github.com/lightning/bolts/pull/798#discussion_r1699198880 ?

Anyway, I like the bolt idea `invoice_signing_pubkey` to make all less confusing.

Reply:
Yeah, though we already use `signing_pubkey` in LDK. Updating the spec would just mean we update the name in `InvoiceTlvStream` and `InvoiceTlvStreamRef`.